UnitedIncome/components
# [APP-3170] Card Illustrations

### Description
> Please provide a brief description of your changes below.

Adds four new circle illustrations.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at them in Storybook.

### Comments
> Any additional comments that will aid with the review process.

There's an issue with the cake illustration's illuminated state, I'll talk to Monique about it before merging.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3170)
